### DISCUSSION TITLE
我想让lobechat 理解我们的业务，应该怎么做

### DISCUSSION THREAD

**Discussion by scortLi** [lobehub/lobehub#1028] [General]
### 🥰 需求描述

我想让lobechat 像Openai 的assistant 一样具有函数调用，代码解释，私有知识库，就是能够理解企业的业务，根据我们的业务回答问题

### 🧐 解决方案

根据企业业务来问答

[a comment by an automated account is not shown]

**Comment by scortLi** [comment]
有没有什么好的解决方案或者有什么思路可以提供吗，谢谢

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
不计划官方支持 Assistant 能力，这对于我们的长期发展来看不太健康。参考：
- https://github.com/lobehub/lobe-chat/discussions/487#discussioncomment-7618927

有两个方案供你参考：

`A.` 等我们实现以下特性：
  - https://github.com/lobehub/lobe-chat/discussions/863
  - 代码解释器插件

`B.` 如果等不及我们，你可以做一个 OpenAI Assistant 的 Lobe Chat 插件，这样可以在这个插件里面使用 assistant 的能力（可以设置插件的 type 为 [markdown](https://chat-plugin-sdk.lobehub.com/zh-CN/guides/markdown-plugin) ），如果对开发流程有疑惑，我们这边可以提供技术支持；

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
我们已经支持知识库功能，可以试试看：https://lobehub.com/zh/blog/knowledge-base
